newsfragments/1003.bugfix

```diff
@@ -0,0 +1 @@
+Add support for attaching a debugger to Go services under telepresence
```

Comment:
Could you please update this since we now know it's not specific to go? 
 
> the-gigi [9:27 PM]
> Also, tested on attaching PyCharm to a swapped Python service. Current telepresence exits immediately when debugger attaches, telepresence+PR works like a [py]charm

😆

Reply:
done